packages/http-server-csharp/src/lib/lib.ts

```diff
@@ -6,11 +6,19 @@ export interface CSharpServiceEmitterOptions {
   /** Choose which service artifacts to emit.  Default is 'all'.*/
   "output-type"?: "models" | "all";
   /** Emit mock implementations of business logic and setup code. Allows the service to respond to requests with mock responses.*/
-  "emit-mocks"?: "none" | "all";
+  "emit-mocks"?: "none" | "no-project" | "all";
```

Comment:
what does `emit-mocks: "no-project"` signify?

Reply:
It emits mocks but no project files.  (config, project, launchsettings, Program.cs startup)

Reply:
would "none" | "mocks-only" | "mocks-and-project-files"  make sense?